docs/index.md

```diff
@@ -73,16 +68,16 @@ This starts a server on port 8080.
 
 #### Running with OpenFin
 
-OpenFin has a command line tool for launching OpenFin applications. This can be installed as follows:
+The ContainerJS project provides an OpenFin-based command line tool, which can be installed as follows:
```

Comment:
You can actually instal `ssf-electron` globally, it's also available via npm, so can be installed globally from there.